card/mod/machines/lib/javascript/decko_mod.js.coffee

```diff
@@ -43,6 +43,9 @@ $(window).ready ->
     else
       item.find('input').val ''
 
+  $('body').on 'click', '.filtered-list-item-delete', ->
+    $(this).closest('li').remove()
+
```

Comment:
this separation was because it's in the pre-filter interface.  happy to move.

Reply:
I'm happy with a move too.